lib/features/manage_account/presentation/mailbox_visibility/mailbox_visibility_view.dart

```diff
@@ -0,0 +1,196 @@
+        color: _responsiveUtils.isWebDesktop(context) ? null : Colors.white,
+        decoration: _responsiveUtils.isWebDesktop(context)
+            ? BoxDecoration(
+                borderRadius: BorderRadius.circular(20),
+                border: Border.all(color: AppColor.colorBorderBodyThread, width: 1),
+                color: Colors.white)
+            : null,
+        child: ClipRRect(
+          borderRadius: BorderRadius.circular(
+              _responsiveUtils.isWebDesktop(context) ? 20 : 0),
+          child: Padding(
+            padding: const EdgeInsets.only(top: 24),
+            child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  if (_responsiveUtils.isWebDesktop(context))...[
+                    const MailboxVisibilityHeaderWidget(),
+                    _buildLoadingView(),
+                    const Padding(
+                      padding:  EdgeInsets.symmetric(vertical: 16),
+                      child:  Divider(color: AppColor.colorDividerMailbox, height: 0.5, thickness: 0.2),
+                    ),
+                  ],
+                  Expanded(child: _buildListMailbox(context)),
+                ]
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildLoadingView() {
+    return Obx(() => controller.viewState.value.fold(
+      (failure) => const SizedBox.shrink(),
+      (success) => success is LoadingState
+        ? const Center(
+          child: Padding(
+            padding: EdgeInsets.only(top: 16),
+            child: SizedBox(
+              height: 24,
+              width: 24,
+              child: CupertinoActivityIndicator(
+                color: AppColor.colorTextButton,
+              ),
+            )),
+        )
+        : const SizedBox.shrink()));
+  }
+
+  Widget _buildListMailbox(BuildContext context) {
+    return SingleChildScrollView(
+        controller: controller.mailboxListScrollController,
+        key: const PageStorageKey('mailbox_list'),
+        physics: const ClampingScrollPhysics(),
+        child: Column(children: [
+          Obx(() => controller.defaultMailboxIsNotEmpty
+              ? _buildMailboxCategory(context, MailboxCategories.exchange, controller.defaultRootNode)
+              : const SizedBox.shrink()),
+          const SizedBox(height: 13),
+          Obx(() => controller.teamMailboxesIsNotEmpty
+              ? _buildMailboxCategory(context, MailboxCategories.teamMailboxes, controller.teamMailboxesRootNode)
+              : const SizedBox.shrink()),
+          const SizedBox(height: 8),
+          Obx(() => controller.personalMailboxIsNotEmpty
+              ? _buildMailboxCategory(context, MailboxCategories.personalMailboxes, controller.personalRootNode)
+              : const SizedBox.shrink()),
+
+        ])
+    );
+  }
+
+  Widget _buildHeaderMailboxCategory(BuildContext context, MailboxCategories categories) {
```

Comment:
this is duplicated?

Reply:
No duplicate, it's different for this screen